app.py

```diff
@@ -35,9 +35,19 @@
 from website import querylog, aws_helpers, jsonbin, translating, ab_proxying, cdn, database
 import quiz
 
+# store map
+from store_map import ad_name2index
+
 # Set the current directory to the root Hedy folder
 os.chdir(os.path.join(os.getcwd(), __file__.replace(os.path.basename(__file__), '')))
 
+LEVEL_MAX_NUM = {
```

Comment:
What is this?

Reply:
The number of adventures per level can be grabbed from the yaml, and beware that they might be different for different languages

Reply:
I don't find the number of adventures per level in coursedata/adventures, I only find that the yaml file is organized by story so that I can't get the number of adventures per level directly.
![image](https://user-images.githubusercontent.com/76264123/142242367-85020589-c01d-4719-88cc-41d0b5a9f11b.png)

Reply:
Yes you will have to loop over the yaml to find the maximum values.